src/docs/product/integrations/aws-lambda/how-it-works.mdx

```diff
@@ -3,17 +3,35 @@ title: Implementation Details
 description: "Understand how the Sentry AWS Lambda integration works unnder the hood"
 ---
 
-
-## How it Works
+## Node:
 
 When Sentry is added to a Lambda function, the following modifications are made to your Lambda functions:
 * The layer for Sentry is added to your Lambdas. Any existing layers are honored and the Sentry layer is appended to the end.
 
    ![](layer.png)
-   
+
 * Sentry adds the following environment variables
   * `NODE_OPTIONS`: This is to preload the `awslambda-auto` module which will automatically initialize Sentry
   * `SENTRY_DSN`: This is set to the DSN of your project
   * `SENTRY_TRACES_SAMPLE_RATE`: This sets the sampling rate for transactions. You can manually edit your environment variables if you want a different sampling rate.
 
   ![](env_variables.png)
+
+
+## Python:
+
+When Sentry is added to a Lambda function, the following modifications are made to your Lambda functions:
+* The layer for Sentry is added to your Lambdas. Any existing layers are honored and the Sentry layer is appended to the end.
+
+   ![](layer_python.png)
+
+  And your Runtime handler will point to Sentry's handler that will initialize the SDK for you, and then call your Runtime handler to invoke your function
+
+   ![](handler_python.png)
+
+* Sentry adds the following environment variables
+  * `SENTRY_INITIAL_HANDLER`: This stores the value of your Runtime handler. If you choose to disable the Sentry layer, this value will be used to set the Runtime handler.
+  * `SENTRY_DSN`: This is set to the DSN of your project
```

Comment:
Add this link for folks who need to better understand the DSN (I'd also suggest adding the link to line 15)
```suggestion
  * `SENTRY_DSN`: This is set to the [DSN](/product/sentry-basics/dsn-explainer/) of your project
```